test/CMakeLists.txt

```diff
@@ -64,30 +64,38 @@ macro(compile_output_test name)
           ${BENCHMARK_CXX_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 endmacro(compile_output_test)
 
+macro(benchmark_add_test)
+  add_test(${ARGV})
+  if(WIN32 AND BUILD_SHARED_LIBS)
+    cmake_parse_arguments(TEST "" "NAME" "" ${ARGN})
+    set_tests_properties(${TEST_NAME} PROPERTIES ENVIRONMENT_MODIFICATION "PATH=path_list_prepend:$<TARGET_FILE_DIR:benchmark::benchmark>")
```

Comment:
is this something users of the library have had to do or are you introducing it with this change? (i don't use windows platforms so i have no idea...)

Reply:
This is exactly the same thing you were previously doing in
https://github.com/google/benchmark/blob/c8ef1ee99eca565f6a58a9f815fba5ef1f210783/.github/workflows/build-and-test.yml#L105-L109

Reply:
(i'm just moving it into the right place)

Reply:
i don't remember adding that.. anyway, it's fine, just seems like an odd windows only thing for cmake to need.